pkg/storage/chunk/schema_caching_test.go

```diff
@@ -24,6 +24,7 @@ func TestCachingSchema(t *testing.T) {
 	baseTime = baseTime.Add(30*24*time.Hour - 1)
 
 	mtime.NowForce(baseTime)
+	defer mtime.NowReset()
```

Comment:
Ok, this was nasty. Could we inject a time interface instead similar to injecting a logger?